platform/podjs/rollup.config.js

```diff
@@ -2,35 +2,40 @@ import commonjs from "@rollup/plugin-commonjs";
 import resolve from "@rollup/plugin-node-resolve";
 import sucrase from "@rollup/plugin-sucrase";
 import json from "@rollup/plugin-json";
+import path from "path";
+
+const pathResolve = (loc) => path.resolve(__dirname, loc);
+
+const nodeModules = "node_modules/**";
 
 export default [
     {
-        input: "src/index.ts",
+        input: pathResolve("src/index.ts"),
         output: [
             {
-                file: "dist/index.es.js",
+                file: pathResolve("dist/index.es.js"),
                 format: "esm",
             },
             {
-                file: "dist/index.js",
+                file: pathResolve("dist/index.js"),
                 format: "cjs",
             },
         ],
         plugins: [
             json(),
             resolve(),
             sucrase({
-                exclude: ["node_modules/**"],
+                exclude: [nodeModules],
```

Comment:
This change could stay, however. I'm all for factoring out to constants repeated literals (all literals, in fact)